wiki/Manual.md

```diff
@@ -2584,6 +2580,22 @@ The `since` argument is an integer UTC timestamp **in milliseconds** (everywhere
 
 If `since` is not specified the `fetchOHLCV` method will return the time range as is the default from the exchange itself.  This is not a bug. Some exchanges will return candles from the beginning of time, others will return most recent candles only, the exchanges' default behaviour is expected. Thus, without specifying `since` the range of returned candles will be exchange-specific. One should pass  the `since` argument to ensure getting precisely the history range needed.
 
+### Get raw OHLCV response
+
```

Comment:
not sure we should add a note that does not apply to other languages, JS is not the most used language

Reply:
added py version. for custom approaches, it would be enough for interested users.